webapp/views/App/views/Designer/SurveyHierarchy/Tree/Tree.js

```diff
@@ -292,11 +292,28 @@ export default class Tree {
       .classed('highlight', true)
   }
 
-  changeNodeDefLabelType(nodeDefLabelType) {
+  _updateLabels() {
     this.svg
       .selectAll('.node-grid')
       .selectAll('a')
-      .text((d) => NodeDef.getLabelWithType({ nodeDef: d.data, lang: this.lang, type: nodeDefLabelType }))
-    this.nodeDefLabelType = nodeDefLabelType
+      .text((d) => NodeDef.getLabelWithType({ nodeDef: d.data, lang: this.lang, type: this.nodeDefLabelType }))
+  }
+
+  get nodeDefLabelType() {
+    return this._nodeDefLabelType
+  }
+
+  set nodeDefLabelType(nodeDefLabelType) {
```

Comment:
is this name right? or should be setNode..

Reply:
uhm, no, it's a setter, and from outside it will be used like tree.nodeDefLabelType = ... 
From the outside it will be similar to set a variable in the class called nodeDefLabelType... but we call _updateLabels() method too.

Reply:
anyway, the name of the setter for "lang" was not correct! it was not consistent with the getter...